docs/release-notes/0.9.0.asciidoc

```diff
@@ -0,0 +1,99 @@
+:issue: https://github.com/elastic/cloud-on-k8s/issues/
+:pull:  https://github.com/elastic/cloud-on-k8s/pull/
+
+[[release-notes-0.9.0]]
+== {n} version 0.9.0
+
+[[feature-0.9.0]]
+[float]
+=== New features
+
+* APM configuration {pull}1181[#1181] (issue: {issue}53[#53])
+* Kibana HTTP certificates {pull}1148[#1148] (issue: {issue}90[#90])
+* Fix status comparison in Openshift {pull}1136[#1136]
+* Utility to dump info to diagnose problems {pull}1115[#1115] (issue: {issue}46[#46])
+* Expose Kibana keystore for secure settings {pull}1063[#1063] (issue: {issue}755[#755])
+* Update base operator Docker image to centos:7 {pull}1043[#1043] (issue: {issue}1042[#1042])
+* Add a custom config parameter to the Kibana spec. {pull}1030[#1030] (issue: {issue}740[#740])
+* Add basic telemetry {pull}1023[#1023]
+* Add support for using custom HTTP certificates in ES. {pull}1013[#1013]
+
+
+
+[[enhancement-0.9.0]]
+[float]
+=== Enhancements
+
+* Ignore resources created in 0.8.x [OPEN] [ISSUE] {pull}1241[#1241]
```

Comment:
Looks like we need to regenerate this just in case you weren't already planning on it

Reply:
Yes will do another run